CMakeLists.txt

```diff
@@ -165,6 +165,12 @@ seastar_generate_ragel(
     IN_FILE redis/protocol_parser.rl
     OUT_FILE ${scylla_gen_build_dir}/redis/protocol_parser.hh)
 
+seastar_generate_ragel(
```

Comment:
for info, change in this file is not absolutely required (only used by IDE)

Reply:
Yes build fully use ninja.
I can remove change in it if you prefer